ext/node/polyfills/02_init.js

```diff
@@ -39,7 +39,7 @@ function initialize(
   // get node's globalThis
   ObjectDefineProperty(globalThis, nodeGlobalThisName, {
     enumerable: false,
-    writable: false,
+    writable: true,
```

Comment:
Mark it as not writable in `99_main.js`

Reply:
I'm actually not sure we can do that, since this property is lazily initialized - ie. it's initialized after `99_main.js` executes.